catalog/spatial/wfs/spatial-wfs-message-api/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/message/api/FeatureTransformationService.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wfs.catalog.message.api;
+
+import ddf.catalog.data.Metacard;
+import java.io.InputStream;
+import java.util.List;
+import java.util.function.BiFunction;
+
+/**
+ * The FeatureTransformationService aggregates the FeatureTransformer services. It splits the given
+ * WFS response into individual FeatureMembers and passes those FeatureTransformer services. It
```

Comment:
❓ It looks like GML 3.1.1 / WFS 1.1.0 supports either multiple "featureMember" elements or a single "featureMemebers" element.  Does this interface support the latter case?